projects/schematics/src/migrations/mechanism/update-ssr/update-ssr-files_spec.ts

```diff
@@ -77,4 +78,20 @@ describe('Update SSR', () => {
       tree
     );
   });
+
+  describe('updateServerFile', () => {
+    it('should rename server.bak to server.ts', async () => {
+      const serverBakPath = `./${SERVER_BAK_FILENAME}`;
+      tree.create(serverBakPath, 'testing');
+      expect(tree.exists(serverBakPath)).toBeTruthy();
+      tree = await updateSsrSchematicRunner.runSchematic(
+        'update-ssr',
+        { name: 'schematics-test' },
+        tree
+      );
+
+      expect(tree.exists(serverBakPath)).toBeFalsy();
+      expect(tree.exists(SERVER_FILENAME)).toBeTruthy();
```

Comment:
would be nice to test whether the `server.ts` file is different after restoration.

Reply:
I've added one more expect. It checks whether the restored server.ts has the same content as server.ts.bak.